bin/build_and_push.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+REGION=us-east-2
```

Comment:
this should probably be configurable via command line args as well for e.g. price reporter extension